package.json

```diff
@@ -20,7 +20,7 @@
     "url": "https://github.com/osartun/Backbone.Undo.js/issues"
   },
   "dependencies": {
-    "backbone": "~1.0.0",
-    "underscore": "~1.4.4"
+    "backbone": "1.0.0 - 1.2.1",
+    "underscore": "1.4.4 - 1.8.3"
```

Comment:
All tests passed for latest versions backbone (1.2.1 now) and underscore (1.8.3 now)
Also it work with older versions.
Then we set range of working versions.